dm/loader/convert_data.go

```diff
@@ -278,7 +282,11 @@ func parseTable(ctx *tcontext.Context, r *router.Table, schema, table, file stri
 			columns = append(columns, col.Name.Name.O)
 		}
 	}
-	if hasGeneragedCols {
+	extendCol, extendVal := r.FetchExtendColumn(schema, table, sourceID)
```

Comment:
seems only need to `FetchExtendColumn` when  `r.TableExtractor !=nil && `rule.SchemaExtractor` !=nil && r.SourceExtractor != nil`, and this can reduce one re match for better performance

Reply:
Since `Extractor` belongs to `TableRule`, it must be matched to get `TableRule`.